data-service/src/main/java/uk/gov/gchq/palisade/service/data/model/AuditErrorMessage.java

```diff
@@ -188,8 +233,7 @@ public int hashCode() {
     @Generated
     public String toString() {
         return new StringJoiner(", ", AuditErrorMessage.class.getSimpleName() + "[", "]")
-                .add("error=" + error)
-                .add(super.toString())
+                .add("error=                " + error)
```

Comment:
No need for all the extra whitespace:
```suggestion
                .add("error=" + error)
```

Reply:
This has been missed

Reply:
done